docs/ml-features.md

```diff
@@ -53,7 +53,10 @@ collisions, where different raw features may become the same term after hashing.
 chance of collision, we can increase the target feature dimension, i.e. the number of buckets 
 of the hash table. Since a simple modulo is used to transform the hash function to a column index, 
```

Comment:
I think we can add it - but we can simply say "The hash function used is MurmurHash 3"